src/components/Articles/Articles.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from "react";
+import { getArticles } from "../../utils/api";
+import './Articles.css'
+
+export default function ViewArticles() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((data) => {
+      setArticles(data);
+    });
+  });
+
+  return (
+    <>
+      {articles.map((article) => {
+        return (
+          <ul key={article.article_id} className="article-list">
```

Comment:
We would like to see an article card component that displays the img title and other properties and would like some more styling too the styling isn't the main priority but some initial styling is definitely recommended